packages/document/main-doc/docs/en/plugin/rsbuild-plugins.mdx

```diff
@@ -0,0 +1,6 @@
+# Overview
+
+Currently, in Modern.js, there are two Rsbuild plugins that are encapsulated as framework plugins:
+
+- [@modern-js/plugin-swc](/plugin/rsbuild-plugins/plugin-swc): Provides SWC compilation support.
```

Comment:
`@modern-js/plugin-swc` is actually a Modern.js plugin, not an Rsbuild plugin, so the title is kind of misleading:

![image](https://github.com/user-attachments/assets/1cd8b777-1ed4-44f4-a605-29964bbf86fd)

Reply:
done~